src/track/bpm.h

```diff
@@ -57,6 +57,19 @@ class Bpm final {
     static int valueToInteger(double value) {
         return std::round(value);
     }
+    static QString roundBpm(double value) {
```

Comment:
The function does not round any value, it only reduces the precision for formatting. Furthermore, the name suggests that it returns a bpm value, not a string. Use a descriptive name like formatBpmAsText or bpmToText.

Reply:
fixed